exercises/matrix/matrix_test.exs

```diff
@@ -0,0 +1,66 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("matrix.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure trace: true, exclude: :pending
+
+defmodule MatrixTest do
+  use ExUnit.Case
+
+  @input "1 2 3\n4 5 6\n7 8 9"
+
+  #@tag :pending
+  test "reading from and writing to string" do
+    matrix = MatrixStruct.from_string(@input)
+
+    assert matrix != nil
```

Comment:
Agreed, and honestly, it doesn't need to even test them on from_string. For this exercise, we could just pass the string to `rows` instead of a `MatrixStruct`, and let them parse it then and return the nested lists. My thought on that was that for typical uses of matrices, you're going to want them to stick around and not re-parse. That led to using a struct, and not wanting to force them to use a particular structure.

That said, I do wind up insisting that no matter the struct, they output lists, so I might as well just have `rows`/`columns` take the string and output the expected row/rows. What would you think about that?